static/app/utils/replays/hooks/useReplayList.tsx

```diff
@@ -27,7 +28,7 @@ function useReplayList({eventView, organization}: Options): Result {
     replays: [],
   });
 
-  const init = useCallback(async () => {
+  const handleFetch = useCallback(async () => {
```

Comment:
handle sounds like it's in response to something, but this method is being commanded to execute the fetch.

maybe like `loadReplays` or idk